.gitignore

```diff
@@ -2,6 +2,7 @@ node_modules
 public/bundle.js
 public/bundle.js.map
 secrets.js
+secrets.txt
```

Comment:
🐼

Reply:
Should probably use secrets.js, or remove secrets.js.